### ISSUE TITLE
Nexus 4 bootloader unlock broken

### ISSUE THREAD

**Issue by NeoTheThird** [MariusQuabeck/magic-device-tool#65]
## Steps to reproduce:

- Clone MDT from github
- `sudo ./launcher.sh`
- `6` for Nexus 4
- `10` for bootloader options
- `2` for Unlock
- Boot Device in fastboot mode
- `Y` to confirm

## What should happen:

Bootloader get's unlocked.

## What does happen:

Syntax Error:

```
./devices/nexus4/android/bootloader.sh: line 90: syntax error near unexpected token `else'
./devices/nexus4/android/bootloader.sh: line 90: `else'
```

**Comment by MariusQuabeck** [comment]
Thanks for reporting this I will check this soon!

**Comment by MariusQuabeck** [comment]
@NeoTheThird please try again

`Detecting device

005f342950526700	fastboot
Device detected !

Follow the instructions on your device

...
OKAY [  6.358s]
finished. total time: 6.358s

Bootloader unlocked

Cleaning up..

Exiting script. Bye Bye`

**Comment by NeoTheThird** [comment]
I think this got fixed some time ago already, sorry, i forgot about updating this issue. Yes, it's working.

**Comment by NeoTheThird** [comment]
Ahh, right, i fixed it in #67. Weird, i assumed github would close this issue automatically then.
